lab2_drones_manager.cpp

```diff
@@ -143,16 +143,19 @@ bool DronesManager::remove_back() {
 }
 
 bool DronesManager::replace(unsigned int index, DroneRecord value) {
```

Comment:
should allot new memory for an object by doing:
DroneRecord* new_value = new DroneRecord(value);